src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable,AreaCalculator {
+    private String color;
+    private double area;
+    private String name;

```

Comment:
As per task description color is the only common property to be used.
area to be calculated with getArea() method dynamically
name is excessive, you may use getClass().getSimpleName() where required
```suggestion
```

Reply:
++